### ISSUE TITLE
Extract image features

### ISSUE THREAD

**Issue by davidnvq** [batra-mlp-lab/visdial-challenge-starter-pytorch#22]
Sorry for open this again. I found that when I use the extracted features (36 features per image) provided by @peteanderson,  it yields lower accuracy on Visdial than the features that you extracted. These features provided by @peteanderson can be generated by using the default configuration and his pretrained model. I expected that these features should be the same as yours but actually they are not. 
![image](https://user-images.githubusercontent.com/30741785/62706328-3d383f80-ba2a-11e9-9ee5-3a70601b9f5f.png)
Could you share some insight which is  super helpful for us to reproduce the result <3 
1. Did you use the default configuration as below? Or did you change some values?
![image](https://user-images.githubusercontent.com/30741785/62706915-65746e00-ba2b-11e9-808a-69816b71c260.png)

2. Which pretrained model did you use? `the pretrained model` or `the alternative pretrained model`:
![image](https://user-images.githubusercontent.com/30741785/62706798-2fcf8500-ba2b-11e9-8000-d988622434c1.png)
Many thanks beforehand.

_Originally posted by @quanguet in https://github.com/batra-mlp-lab/visdial-challenge-starter-pytorch/issues/15#issuecomment-519516973_
